packages/expect/src/fakeChalk.ts

```diff
@@ -3,12 +3,11 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- * @flow
  */
 
 import ansiStyles from 'ansi-styles';
 
-const returnInput = str => str;
+const returnInput = (str: any) => str;
```

Comment:
```suggestion
const returnInput = (str: string) => str;
```

? Based in the name of the variable

Reply:
Oh makes sense